ood-portal-generator/templates/ood-portal.conf.erb

```diff
@@ -185,6 +185,10 @@ Listen <%= addr_port %>
   SetEnv OOD_PUN_PRE_HOOK_EXPORTS "<%= @pun_pre_hook_exports %>"
   <%- end -%>
 
+  <%- end -%>
+  <%- if @rails_config_hosts -%>
+  SetEnv OOD_PUN_RAILS_CONFIG_HOSTS "<%= @rails_config_hosts.join(',') %>"
```

Comment:
I can't if this will actually make it to the PUN without testing. Also I'm bogged down with some OSCUS stuff so there may be some delay.

Reply:
@johrstrom This PR as is does not work.  What is correct way to take my local code and make a dev container?  I tried `dev:rebuild` and `package:dev_container` and it seemed like both just installed from nightly and didn't use the code I was working on.  

So far what I ended up doing was copying my changes into container with small sinatra app to dump data and seems OOD_PUN_RAILS_CONFIG_HOSTS is not getting set like I'd expect.  I'm not sure why this is the case but if I had to guess the Apache SetEnv isn't same as bash env used to launch PUN.

Reply:
Setting these `OOD_MNT_*` env variables will mount your local update directory into the container.

https://github.com/OSC/ondemand/blob/master/DEVELOPMENT.md#additional-mounts

You can set them to anything, as long as they're not nil.

`export OOD_MNT_PORTAL='yesplz'`

Reply:
>  I'm not sure why this is the case but if I had to guess the Apache SetEnv 

I don't think that's a real ENV variable. I think it's just an Apache construct, that's why we had to add `lua-posix` to set real environment variables.

Reply:
I fixed the issue.  Had to modify mod_ood_proxy to also ensure the environment variable was passed from Apache to the shell executing the PUN startup.

Reply:
My sinatra app did this:

```
  puts "USER #{ENV['USER']} TEST #{ENV['OOD_PUN_RAILS_CONFIG_HOSTS']} ENV=#{ENV.inspect}"
```

I got this (ENV dump not pasted)

```
USER tdockendorf TEST localhost ENV=....
```